hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/HoodieTableFactory.java

```diff
@@ -246,6 +246,8 @@ private static void setupConfOptions(
       ResolvedSchema schema) {
     // table name
     conf.setString(FlinkOptions.TABLE_NAME.key(), tablePath.getObjectName());
+    // database name
+    conf.setString(FlinkOptions.DATABASE_NAME.key(), tablePath.getDatabaseName());
     // hoodie key about options
```

Comment:
we need some test for it. like in `TestHoodieTableFactory`.

Reply:
Sure! test added in `TestHoodieTableFactory`